docs/guides/bake/configuring-build.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Configuring builds"
+keywords: build, buildx, bake, buildkit, hcl, json
+---
+
```

Comment:
This should start with an intro, e.g. "Bake supports loading build definition from files. But sometimes you need even more flexibility to configure this definition. For this use case, you can define variables inside the bake files that can be set by the user with environment variables or by attribute definitions in other bake files. If you wish to change a specific value for a single invocation you can use a `--set` flag."

Reply:
Added an intro and also moved some bits from HCL user-defined funcs specific to build configuration in this page.